Makefile.js

```diff
@@ -550,8 +550,11 @@ target.browserify = function() {
     // 6. Browserify espree
     nodeCLI.exec("browserify", "-r espree", "-o", TEMP_DIR + "espree.js");
 
-    // 7. Concatenate the two files together
-    cat(TEMP_DIR + "espree.js", BUILD_DIR + "eslint.js").to(BUILD_DIR + "eslint.js");
+    // 7. Browserify astUtils as its needed by eslint tests
```

Comment:
This shouldn't be necessary. As long as it's required, it should show up automatically.